<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OneDriveSettingsWidget</class>
 <widget class="QWidget" name="OneDriveSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OneDrive Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="oneDriveEnable">
     <property name="text">
      <string>Enable OneDrive streaming</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="authenticationGroup">
     <property name="title">
      <string>Authentication</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="authStatusLabel">
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="authStatus">
        <property name="text">
         <string>Not configured</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="authenticateButton">
        <property name="text">
         <string>Configure Authentication...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="testConnectionButton">
        <property name="text">
         <string>Test Connection</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="cacheGroup">
     <property name="title">
      <string>Cache Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="cacheSizeLabel">
        <property name="text">
         <string>Cache Size (MB):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="cacheSize">
        <property name="minimum">
         <number>16</number>
        </property>
        <property name="maximum">
         <number>2048</number>
        </property>
        <property name="value">
         <number>64</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="prefetchSizeLabel">
        <property name="text">
         <string>Prefetch Size (MB):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="prefetchSize">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>256</number>
        </property>
        <property name="value">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="clearCacheButton">
        <property name="text">
         <string>Clear Cache</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="requirementsGroup">
     <property name="title">
      <string>Bandwidth Requirements</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="bandwidthInfo">
        <property name="text">
         <string>Loading...</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>